goa/proj.android-studio/goa/src/main/java/org/cocos2dx/cpp/AppActivity.java

```diff
@@ -370,6 +371,16 @@ public void run() {
         }
     }
 
+    int mCoins;
+
+    public int getCoins() {
+        return mCoins;
+    }
+
+    public void setCoins(int mCoins) {
```

Comment:
Check where this is being used. 
If required, move this to the appropriate place